MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -46310,10 +46497,39 @@
     {
       "Id": "CollectableWoodfallItem1",
       "RequiredItems": [
-        "AreaSouthAccess",
-        "Woodfall Platforming"
+        "AreaSouthAccess"
+      ],
+      "ConditionalItems": [
+        [
+          "MaskDeku"
+        ],
+        [
+          "FD Difficult Jumps"
```

Comment:
Same question about Woodfall Platforming

Reply:
Same answer as before, I deleted Woodfall Platforming to rewrite every check in Woodfall individually.